lastuser_oauth/views/login.py

```diff
@@ -172,6 +173,9 @@ def reset():
     if getbool(request.args.get('expired')):
         message = _(u"Your password has expired. Please enter your username "
             "or email address to request a reset code and set a new password")
+    elif getbool(request.args.get('nopasswd')):
+        message = _(u"Your account does not have a password set. Please enter your username "
+            "or email address to request a reset code and set a new password")
```

Comment:
Remove this and just depend on the flash message. Remove the `expired` check as well unless there's an email that goes out when a password expires (can't use flash messages there).